Source/WebCore/platform/mediastream/gstreamer/GStreamerVideoCapturer.cpp

```diff
@@ -82,9 +82,6 @@ GstElement* GStreamerVideoCapturer::createConverter()
         m_videoSrcMIMETypeFilter = gst_element_factory_make("capsfilter", "mimetype-filter");
         head = m_videoSrcMIMETypeFilter.get();
 
-        auto caps = adoptGRef(gst_caps_new_empty_simple("video/x-raw"));
-        g_object_set(m_videoSrcMIMETypeFilter.get(), "caps", caps.get(), nullptr);
```

Comment:
If we never set caps on this the whole element is kinda useless.

Reply:
Isn't it getting set [here](https://github.com/WebKit/WebKit/blob/92cc4d5d99140da8ed1154a4eb309eba647841d9/Source/WebCore/platform/mediastream/gstreamer/GStreamerVideoCapturer.cpp#L359) as well?

Reply:
Best I can tell setting `video/x-raw` here doesn't make much sense as a plain `video/x-raw` mime-type isn't even valid without a format specifier from my understanding.

Reply:
Removed this part for now since it's not really clear what the correct fix is.